### ISSUE TITLE
Python path not being recognized

### ISSUE THREAD

**Issue by rishikanthc** [mtshiba/pylyzer#109]
I just installed pylyzer by building it from source. Resolving local imports doesn't seem to work even after setting python path in pyproject.toml. Below is my `pyproject.toml`.

```toml
[tool.poetry]
name = "hdc"
version = "0.1.0"
description = ""
authors = ["Your Name <you@example.com>"]
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.12"
loguru = "*"
numpy = "*"
einops = "*"

[tool.poetry.group.dev.dependencies]
ruff = "*"
pre-commit = "*"
pytest = "*"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"

[tool.pylyzer.python]
path = "/Users/rishi/miniconda3/envs/hdc/bin/python"
```

For eg. when I try to import `loguru` I get a `module not found` error.

**Comment by mtshiba** [comment]
Support for virtual environments has been implemented in v0.0.66.
We have confirmed that it works with uv/poetry (no additional settings are required).

**Comment by evbo** [comment]
@mtshiba can we keep this issue open until it's confirmed working with more than just uv or poetry?

I have a plain setuptools project with pyproject.toml set. Mypy had no issue with it but pylyzer is unusable. I don't even know where to begin as it seems that setuptools projects are not in scope.
